src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceSyncResponse.scala

```diff
@@ -14,21 +16,25 @@ object InstanceSyncResponse {
   implicit val instanceSyncResponseDecoder: Decoder[InstanceSyncResponse] = (c: HCursor) =>
     for {
       nextCursor <- c.downField("nextCursor").as[Option[Map[String, String]]]
-      items <- c
+      typing <- c
+        .downField("typing")
+        .as[Option[Map[String, Map[String, Map[String, Map[String, TypePropertyDefinition]]]]]]
+      itemObjects <- c
         .downField("items")
-        .as[Option[Map[String, Seq[InstanceDefinition]]]](
-          Decoder.decodeOption[Map[String, Seq[InstanceDefinition]]](
-            Decoder.decodeMap[String, Seq[InstanceDefinition]](
-              implicitly,
-              Decoder.decodeIterable[InstanceDefinition, Seq](
+        .as[Option[Map[String, Seq[Json]]]]
```

Comment:
will need to check error handling below, have a feeling it might be exposing parse errors differently than before

Reply:
yep, `.as[...].toOption` was hiding parse errors, fixed now via use of `traverse` to work with eithers deep inside the result